interface/main/tabs/main.php

```diff
@@ -432,5 +405,4 @@ function showUpdaterNotifications(type, title, message) {
         });
     }
 }
-</script>
-
+</script>
```

Comment:
Don't delete all whitespace from files.  It is used for code visibility.  Also, your editor is deleting the line return at the end of this file.  A line return (blank line) is needed for GIT file end normalization and helps the repo system in some mysterious way I think I actually understood at some point.